Packs/Stealthwatch_Cloud/Integrations/Stealthwatch_Cloud/Stealthwatch_Cloud.py

```diff
@@ -55,7 +55,7 @@ def http_request(method, url_suffix, params_dict=None, headers=DEFAULT_HEADERS,
         except ValueError:
             # in case the response doesn't have JSON
             return "Request completed"
-    except Exception, e:
+    except Exception as e:
```

Comment:
This is python 2 integration, not python 3

Reply:
This syntax works in BOTH.